lib/log_parser.rb

```diff
@@ -19,31 +17,48 @@ def first_line
 
   def gamelog_json
     @gamelog_obj_ruby = {
-      @file_name => {
-        lines: line_count(),
-        players: [],
+      @file_name.to_sym => {
+        lines: @data.size(),
+        players: player_count(),
+        kills: kills_per_player(),
+        total_kills: total_kills_count()
       } 
     }
-    player_count
-    gamelog_obj_json = @gamelog_obj_ruby.to_json
+    gamelog_obj_json = JSON.pretty_generate(@gamelog_obj_ruby)
   end
 
   private
   
-  def line_count
-    @lineCount = @data.size
-    result
+  def kills_per_player
+    killer_numbers = []
+    File.foreach(@file_path) do |line|
```

Comment:
Da um espacinho entre um bloco, facilita a leitura :D
```suggestion

    File.foreach(@file_path) do |line|
```